core/lib/spree/testing_support/sequences.rb

```diff
@@ -0,0 +1,9 @@
+require 'factory_girl'
```

Comment:
I wouldn't mind doing that, but that was Kevin's decision, and I didn't want to touch his stuff. @Sinetheta should the file be moved?